lib/manager/gradle/index.js

```diff
@@ -0,0 +1,136 @@
+const childProcess = require('child_process');
+const fs = require('fs');
+const path = require('path');
+
+const gradle = require('./build-gradle');
+
+const GRADLE_DEPENDENCY_REPORT_COMMAND =
+  'gradle --init-script init.gradle dependencyUpdates -Drevision=release';
+const GRADLE_DEPENDENCY_REPORT_FILENAME = 'build/dependencyUpdates/report.json';
+
+function extractDependencies(content, fileName, config) {
+  logger.debug('gradle.extractDependencies()');
+  const gradleFile = path.join(config.localDir, fileName);
+
+  fs.writeFileSync(gradleFile, content);
+  configureUseLatestVersionPlugin(config.localDir);
+  const gradleSuccess = executeGradle(config);
+
+  if (gradleSuccess) {
+    const deps = extractDependenciesFromUpdatesReport(config.localDir);
+    if (deps.length === 0) {
```

Comment:
Now it returns the project dependencies (all of them) or null if there is no any; which will never happens because the useLastVersionPlugin is counted as one dependency.

Reply:
Can we exclude it from the count? I think it's better for users to ignore the file completely if it has no genuine deps

Reply:
Yep, I think it should be more or less easy. Only thing I need to detect is if the user has the plugin configured by himself because in that case it would be a genuine dependency.